doc/man1/openssl-s_client.pod.in

```diff
@@ -781,7 +781,7 @@ server.
 
 This command is a test tool and is designed to continue the
 handshake after any certificate verification errors. As a result it will
-accept any certificate chain (trusted or not) sent by the peer. None test
+accept any certificate chain (trusted or not) sent by the peer. Non-test
```

Comment:
Rereading the sentence I get the impression that this little change does not really improve it. How about reformulating the sentence, for example as follows:

```
Applications should B<not> do this in productive use as it makes them vulnerable...
```

Reply:
Or perhaps

> This is only appropriate in a testing tool; other applications need to perform certificate validation and engage error handling on validation failure.  Failure to perform proper validation leaves the application vulnerable to a MITM attack.

Though I'm willing to accept the current version of this patch as a clear improvement over the existing buggy state.

Reply:
The latter was my intention, but I'm also happy to close this pull request if someone else comes up with a more substantial improvement.